projects/prison-case-notes-to-probation/src/test/kotlin/uk/gov/justice/digital/hmpps/messaging/HandlerTest.kt

```diff
@@ -99,8 +101,12 @@ internal class HandlerTest {
         verify(telemetryService).trackEvent(eq("CaseNoteIgnored"), any(), any())
     }
 
-    @Test
-    fun `when ocg alert text has been sent`() {
+    @ParameterizedTest
+    @ValueSource(
```

Comment:
Nice! I didn't know about the `@ValueSource` annotation, looks like it makes parameterized tests a bit simpler to write.